src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTab] = useState(tabs[0]);
```

Comment:
It's not really `tabId` but just whole `tab`.
So you can name it as `selectedTab`.